src/objects/core/zif_abapgit_function_module.intf.abap

```diff
@@ -0,0 +1,11 @@
+INTERFACE zif_abapgit_function_module
+  PUBLIC.
+
+  METHODS:
+    function_exists
+      IMPORTING
+        iv_function_module_name TYPE rs38l-name
```

Comment:
anything released that can be used instead of `rs38l-name`? https://abapedia.org/steampunk-2305-api/, perhaps https://abapedia.org/steampunk-2305-api/d_delfunc.dtel.html

perhaps just make it a string, and convert in the implementation?

Reply:
done